examples/nlp/multi_label_classification.py

```diff
@@ -145,7 +146,7 @@
 """
 
 terms = tf.ragged.constant(train_df["terms"].values)
```

Comment:
Would it work to use padding here instead of ragged tensors?
https://keras.io/api/ops/numpy/#pad-function